### ISSUE TITLE
va-textarea extends outside its container at high zoom

### ISSUE THREAD

**Issue by amyleadem** [department-of-veterans-affairs/vets-design-system-documentation#4799]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened
In high-zoom environments (400% at 1280px or 320px), `va-textarea` gets cut off because it is wider than the viewport. This causes the focus outline to be cut off as well. This happens in both Storybook and production environments.

Originally reported in issue https://github.com/department-of-veterans-affairs/va.gov-team/issues/107758

<img width="1283" height="704" alt="Image" src="https://github.com/user-attachments/assets/2f36ac2f-d0b7-4f38-a84e-02b01ef80ed0" />

## What I expected to happen

At 400% zoom, the component should be fully visible without requiring horizontal scroll.

## Reproducing

- Device: Mac
- Browser: Chrome

**Steps to reproduce:**

1. Open [va-textarea in Storybook](https://design.va.gov/storybook/iframe.html?globals=&args=&id=uswds-va-textarea--default&viewMode=story)
2. Set the viewport to 1280px and zoom in 400% 
3. Notice that the component extends beyond the viewport

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [ ] Other

## Possible solution
This can likely be fixed by adding `box-sizing: border-box` to the component styles. Consider importing inputs.css (`@import '../../mixins/inputs.css'`) to the component stylesheet.
